angular-ngrx-scss/src/app/user/services/user.service.ts

```diff
@@ -105,4 +108,49 @@ export class UserService {
       ),
     );
   }
+
+  getUserTopRepos(): Observable<UserReposState[]> {
+    const url = `${environment.githubUrl}/user/repos?sort=updated&affiliation=owner,collaborator,organization_member&per_page=20'}`;
```

Comment:
suggestion: GET params should be included using the params object that the HTTP service accepts. I would consider implementing params in a manner similar to [this snippet](https://github.com/thisdot/starter.dev-github-showcases/blob/8648d0c18a87e61c460cbcf552ba5d9fd5ffddaf/angular-ngrx-scss/src/app/organization/services/organization.service.ts#L16-L40).

You could forgo accepting the params as a parameter in the function since this call is ad-hoc, but I would still recommend keeping these values in a map so they are easier to read and edit. You could pass a literal map to the `params` key in this case and that should work. You do not need to define an interface in this context like in the aforementioned example if I recall correctly.